cou/apps/auxiliary.py

```diff
@@ -351,7 +351,7 @@ async def _verify_nova_compute(self, target: OpenStackRelease) -> None:
         units_not_upgraded = []
         apps = await self.model.get_applications()
 
-        for app in apps:
+        for app in apps.values():
```

Comment:
Why we are not checking only the app itself? Why all nova-compute's in model.

Reply:
Ach yes, this is in caph-osd. I thought it is in nova-compute app. Sorry and thanks for replay.